Compilation fails on appveyor

```
lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(127): error C3861: '_mm_srli_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(128): error C3861: '_mm_srli_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(138): error C3861: '_mm_sub_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(139): error C3861: '_mm_sub_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(140): error C3861: '_mm_cvtpi32x2_ps': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(141): error C3861: '_mm_empty': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(234): error C3861: '_mm_cvttps_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(235): error C3861: '_mm_cvttps_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(237): error C3861: '_mm_cvtpi32x2_ps': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(272): error C3861: '_mm_cvttps_pi32': identifier not found 
c:\miniconda3-x64\conda-bld\work\msmbuilder\hmm\src\include\sse_mathfun.h(273): error C3861: '_mm_cvttps_pi32': identifier not found 
c:\miniconda3-x6
```

--- a/msmbuilder/hmm/src/include/sse_mathfun.h
+++ b/msmbuilder/hmm/src/include/sse_mathfun.h
@@ -124,6 +124,10 @@ static v4sf log_ps(v4sf x) {
 #ifndef USE_SSE2
   /* part 1: x = frexpf(x, &e); */
   COPY_XMM_TO_MM(x, mm0, mm1);
+  On appveyor, these functions arent defined
+  You should probably be using SSE2. If you really want
+  MMX, delete these lines (which trigger a syntax error even
+  on platforms that support MMX)
   mm0 = _mm_srli_pi32(mm0, 23);
   mm1 = _mm_srli_pi32(mm1, 23);
 #else

--- a/msmbuilder/hmm/src/logsumexp.hpp
+++ b/msmbuilder/hmm/src/logsumexp.hpp
@@ -1,5 +1,5 @@
 /*****************************************************************/
-/*    Copyright (c) 2013, Stanford University and the Authors    */
+/*    Copyright (c) 2016, Stanford University and the Authors    */
 /*    Author: Robert McGibbon <rmcgibbo@gmail.com>               */
 /*    Contributors:                                              */
 /*                                                               */
@@ -11,6 +11,9 @@
 #include "float.h"
 #include <stdio.h>
 #include <math.h>
+
+// Please define USE_SSE2 before using sse_mathfun.h
+#define USE_SSE2
 #include "sse_mathfun.h"
 
 #if defined(_MSC_VER)

